ThirdPartyAdapters/vungle/vungle/build.gradle

```diff
@@ -36,8 +36,13 @@ android {
 }
 
 dependencies {
-    implementation ('com.vungle:publisher-sdk-android:6.5.1') {
-        transitive=true
+    if (gradle.ext.has('dependOnVungleSdkProject') && gradle.ext.dependOnVungleSdkProject) {
+        implementation project(':vungle-android-sdk:publisher-sdk-android')
+    } else {
+        implementation ('com.github.vungle:vungle-android-sdk:6.5.2-RC3') {
```

Comment:
can we bump this to RC4 , thats our latest with some fixes for MOAT's new View ability 3rd party lib

Reply:
fixed